components/com_kunena/site/template/crypsis/layouts/bbcode/map/default.php

```diff
@@ -24,7 +24,7 @@
 	$id = 0;
```

Comment:
Please update this to (New look/api, code still work): 
$this->addScript('http://maps.google.com/maps/api/js?v=3.exp&signed_in=true');
$id = 0;